app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/repository/RepositoryFragmentViewModel.kt

```diff
@@ -50,16 +48,20 @@ class RepositoryFragmentViewModel(
             it.workspace?.slug?.let { workspaceSlug ->
                 it.name?.let { repoName ->
                     launchIO {
-                        val result = repo.getSource(workspaceSlug, repoName)
-                        when (result) {
-                            is ApiResult.Success -> srcFiles.set(result.data)
-                            is ApiResult.Failure -> {
-                                // TODO: Improve error messaging
-                                withContext(dispatcherProvider.Main) {
-                                    toaster.toast(R.string.error_loading_repository)
-                                }
-                            }
-                        }.exhaustive
+                        // val result = repo.getSource(workspaceSlug, repoName)
+                        // when (result) {
+                        //     is ApiResult.Success -> srcFiles.set(result.data)
+                        //     is ApiResult.Failure -> {
+                        //         TODO: Improve error messaging
+                        // withContext(dispatcherProvider.Main) {
+                        //     toaster.toast(R.string.error_loading_repository)
+                        // }
+                        // }
+                        // }.exhaustive
```

Comment:
@BottleRocket-Colin nit: Delete these commented lines

Reply:
I thought I reverted those out :(

Reply:
Should be fine. Can get them removed in the next one :D